src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -1442,6 +1442,29 @@ def load_arguments(self, _):
         c.argument('enable_auto_key_rotation', arg_type=get_three_state_flag(), min_api='2020-12-01',
                    options_list=['--enable-auto-key-rotation', '--auto-rotation'],
                    help='Enable automatic rotation of keys.')
+
+    with self.argument_context('disk-encryption-set create', operation_group='disk_encryption_sets',
+                               min_api='2022-03-02') as c:
+        c.argument('federatedClientId', help='The federated client id used in cross tenant scenario.')
```

Comment:
Hump is not recommended in our parameter naming style, so it is recommended to name it `--federated-client-id` to keep consistent with the style of other parameters. @laurawu19 Please confirm it~
```suggestion
        c.argument('federated_client_id', help='The federated client id used in cross tenant scenario.')
```